### DISCUSSION TITLE
webp image support

### DISCUSSION THREAD

**Discussion by takeshi-codes** [joemasilotti/railsdevs.com#784] [Ideas]
Currently, when trying to upload a .webp image file for either your cover photo or profile picture, you will get an error stating that the format isn't supported.

I'm not sure if this is on the roadmap, or even been thought about. I'd be happy to work on a PR for it if its not already in the works.

**Comment by joemasilotti** [comment]
Not all browsers can display .webp, right? So introducing this as an input would require processing on the images to make sure all browsers can render them when being displayed.

If so, I don't think I want to go down that path. Unless we can consolidate all of the image processing to a single filetype without much code to maintain.

**Reply by takeshi-codes** [reply]
Oh! Thanks for the feedback, I was unaware not all browsers supported .webp. not my project but I agree with the decision, I'll close this out or whatever I gotta do.

**Reply by jkostolansky** [reply]
Which browser doesn't support webp format? [Can I use](https://caniuse.com/webp) says only IE.

**Reply by joemasilotti** [reply]
TIL! Thanks for digging that up @jkostolansky.

@thisisdylandev, do you still want to tackle this issue?

**Reply by takeshi-codes** [reply]
@joemasilotti unfortunately I no longer have the bandwidth to pick up some extra work
